app/src/main/java/com/davidtomas/taskyapp/navigation/MainActivityViewModel.kt

```diff
@@ -34,6 +38,8 @@ class MainActivityViewModel(
                 }
             )
             _isAuthChecked.value = true
+            delay(2500L)
+            _isSplashFinished.value = true
```

Comment:
Why this delay? The splash screen should hide as soon as you know where to navigate. Artificially delaying it is not a good UX